user_guide.adoc

```diff
@@ -1377,30 +1377,82 @@ library" for expression language.
 |Return type
 |Description
 
-|to_s
+|`to_s`
 |String
 |Converts integer into a string using decimal representation
-
-|to_s(radix)
```

Comment:
To clarify the confusion, this comment thread refers to methods which can be called on IntType, not StrType.